devops/connectors/Dockerfile

```diff
@@ -1,14 +1,14 @@
 # Context - root of java project
 # Build Stage
-FROM cm/java-base:1.0.0 AS build
-COPY . /usr/src/project
-WORKDIR /usr/src/project
+FROM openjdk:8u131-jdk AS build
+COPY ./ /cm
+WORKDIR /cm
 RUN ./gradlew clean :connectors:installDist
 
 # Container with application
 FROM openjdk:8u131-jre-alpine
-WORKDIR /usr/src/connector
-COPY --from=build /usr/src/project/connectors/build/install/connectors .
+COPY --from=build /cm/connectors/build/install/connectors /cm/connectors/dist
+WORKDIR /cm/connectors/dist
 CMD ["./bin/connectors"]
```

Comment:
Single `connectors` binary that does all things at once doesn't seem like a good idea to me.

We use convention to move binaries to `/cyberapp`, CMD to ENTRYPOINT, so that we can add more arguments to command line.

Also, no WORKDIR, because containers supposed to be read-only. All data should be written into `VOLUME /cyberdata`, which is mounted for each container separately. Let me know if this setup creates any problems for you.